_apps/create-issue-branch.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Create Issue Branch
+description: Creates a new branch after assigning an issue
+slug: create-issue-branch
+screenshots:
+- https://github.com/robvanderleek/create-issue-branch/blob/master/static/create-issue-branch.gif?raw=true
+authors: [robvanderleek]
+repository: https://github.com/robvanderleek/create-issue-branch
```

Comment:
repository should be without `https://github.com/`

Reply:
Ah, thanks, fixed that.